pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -272,7 +273,10 @@ func (q *Queue) waitOrTerminate(ctx context.Context, cmd *Command) error {
 			metrics.NodeClaimsTerminatedCounter.With(prometheus.Labels{
 				metrics.ReasonLabel:       cmd.method,
 				metrics.NodePoolLabel:     cmd.candidates[i].NodeClaim.Labels[v1beta1.NodePoolLabelKey],
+				metrics.ZoneLabel:         cmd.candidates[i].NodeClaim.Labels[v1.LabelTopologyZone],
+				metrics.ArchLabel:         cmd.candidates[i].NodeClaim.Labels[v1.LabelArchStable],
```

Comment:
Both of these labels IIRC are on kube_node_info. Zone and arch are relatively low cardinality though so not the end of the world.

 KSM has most of these values in one way or another that if your cluster is low enough scale you can scrape the node labels metrics and get this information.  Is there a reason you need them on the termination counter metrics? It is nice to have that preassigned grouping by nodepool. 
 
 
 Overall would say I am in favor of Zone and Arch but not InstanceType, would be good to get the opinion of other maintainers